cognee/infrastructure/databases/vector/lancedb/LanceDBAdapter.py

```diff
@@ -72,6 +72,49 @@ class LanceDataPoint(LanceModel):
                 exist_ok = True,
             )
 
+
+
+    async def create_relationships(self, collection_name: str, relationships: list[Relationship]):
+        """Create and store Relationship embeddings in LanceDB."""
+        connection = await self.get_connection()
+
+        # Ensure collection exists
+        if not await self.has_collection(collection_name):
+            await self.create_collection(collection_name, Relationship)
+
+        collection = await connection.open_table(collection_name)
+
+        # Generate embeddings
+        data_vectors = await self.embed_data([
+            " ".join([str(v) for v in rel.get_embeddable_properties().values()])
+            for rel in relationships
+        ])
+
+        # Dynamic LanceDataPoint class for Relationship
+        vector_size = self.embedding_engine.get_vector_size()
+
+        class LanceRelationship(LanceModel):
+            id: str
+            vector: Vector(vector_size)
+            payload: dict
+
+        # Prepare LanceDB-compatible data points
+        lance_relationships = [
```

Comment:
I think for this, we have the index datapoints method, which indexes the datapoint pydantic object based on its index_fields property.